src/auth/basic/LDAP/Makefile.am

```diff
@@ -7,10 +7,10 @@
 
 include $(top_srcdir)/src/Common.am
 
-libexec_PROGRAMS 	= basic_ldap_auth
-man_MANS 		= basic_ldap_auth.8
-EXTRA_DIST		= basic_ldap_auth.8 required.m4
-basic_ldap_auth_SOURCES	= basic_ldap_auth.cc
+libexec_PROGRAMS = basic_ldap_auth
```

Comment:
How are these lines being changed? I do not see the same whitespace removals in other files using this indentation style.

Reply:
That was a manual change on my side. For some reason these lines use tabs instead of spaces; I've converted them to use spaces for consistency.

Reply:
This is just one of the helper Makefile.am using this indentation with tabs for alignment. Basic auth LDAP, NCSA, PAM, RADIUS helper to be specific.

If one gets cleaned they all should. But really this change is out of scope for a sorting PR.